packages/exo/test/test-heap-classes.js

```diff
@@ -203,6 +203,53 @@ test('sloppy option', t => {
   );
 });
 
+const RawGreeterI = M.interface('greeter', {}, { defaultGuards: 'raw' });
+
+const testGreeter = (t, greeter, msg) => {
+  const mutable = {};
+  t.is(greeter.sayHello(mutable), 'hello', `${msg} can sayHello`);
+  t.deepEqual(mutable, { x: 3 }, `${msg} mutable is mutated`);
+  mutable.y = 4;
+  t.deepEqual(mutable, { x: 3, y: 4 }, `${msg} mutable is mutated again}`);
+};
+
+test('rawValues', t => {
+  const greeter = makeExo('greeter', RawGreeterI, {
+    sayHello(mutable) {
+      mutable.x = 3;
+      return 'hello';
+    },
+  });
+  t.deepEqual(greeter[GET_INTERFACE_GUARD](), RawGreeterI);
+  testGreeter(t, greeter, 'raw defaultGuards');
+
+  const Greeter2I = M.interface('greeter2', {
+    sayHello: M.call(M.rawValue()).returns(M.string()),
```

Comment:
I'm new to the motivating issues and don't grok them yet. "It would be nice for even finer granularity" why?

Here it's not clear to me how the behavior of `M.rawValue()` is different than `M.any()`. Please include a test contrasting them specifically.

Reply:
I'll include more explicit tests.

As to the difference: matching `M.any()` automatically hardens its input, then enforces it is Passable (e.g. is compatible with inter-vat marshalling).  `M.rawValue()` avoids that processing: it allows designated arguments or return values through verbatim.

Reply:
I'll include more explicit tests.

As to the difference: matching `M.any()` automatically hardens its input, then enforces it is Passable (e.g. is compatible with inter-vat marshalling).  `M.rawValue()` avoids that processing: it allows designated arguments or return values through verbatim.

Reply:
Ah, I didn't realize:
- the matchers do modify the argument (hardening)
- M.any() doesn't really alllow _any_; it requires Passable. I think it's a good name but maybe some jsdoc is in order: "any Passable"

Given that the other matchers are for values, why should "Value" be included in the method name? iow, why not `M.raw()` ?

Reply:
"even finer granularity" is what this PR provides.  The issue that mentioned the need was addressed at an older PR that only did interface-granularity.

`M.rawValue()` doesn't do any processing of the designated argument or return value.  OTOH, `M.any()` automatically hardens the designated value, and asserts that it is Passable (i.e. can be marshalled using `@endo/marshal`).

I'll add some more specific tests.

Reply:
>M.rawValue() doesn't do any processing of the designated argument or return value. OTOH, M.any() automatically hardens the designated value, and asserts that it is Passable (i.e. can be marshalled using @endo/marshal).

I got that in the previous message. 

When I said "maybe some jsdos is in order: 'any Passable'" I meant to make it more clear that M.any doesn't match "any value" but any _passable_. It turns out that's already in the jsdoc,
https://github.com/endojs/endo/blob/54ff59f9e44a19a4ef7c9d3f72b01df8c9059626/packages/patterns/src/types.js#L239-L241

However, none of those JSDocs indicate that the argument will be hardened. That seems important to say. Other APIs complain when the argument isn't already hardened so this seems inconsistent and likely to surprise people. Not something that has to be solved in this PR, of course.

I don't see an answer in your reply to this question:
> Given that the other matchers are for values, why should "Value" be included in the method name? iow, why not `M.raw()` ?

Reply:
> I got that in the previous message.

Yeah, as discussed offline I thought my phone lost my first comment (when it actually posted it twice), so from-memory recapitulation probably obscured what I was trying to say.

> However, none of those JSDocs indicate that the argument will be hardened. That seems important to say. Other APIs complain when the argument isn't already hardened so this seems inconsistent and likely to surprise people. Not something that has to be solved in this PR, of course.

What I didn't state correctly is that `M.any()` doesn't actually do the hardening.  It's the default behaviour of `M.call()` or `M.callWhen()` that hardens arguments and return values before matching patterns, and `M.rawValue()` opts out of this behaviour.  I will add some JSDoc to `.call` and `.callWhen` to make this clearer.
 
> > Given that the other matchers are for values, why should "Value" be included in the method name? iow, why not `M.raw()` ?

I spoke with @erights, and we agreed that `M.raw()` would be better.  I'll make that change.